app/views/layouts/accessible_format_requests.html.erb

```diff
@@ -10,6 +10,9 @@
   </head>
   <body>
     <div id="wrapper" class="govuk-width-container">
+    <% if @back_path %>
+      <%= render "govuk_publishing_components/components/back_link", { href: @back_path } %>
+    <% end %>
```

Comment:
There may be an issue with moving the "Back" link to the layout view as it creates a larger gap between the link and the page title.

<table role="table">
<thead>
<tr>
<th>Before</th>
<th>After</th>
</tr>
</thead>
<tbody>
<tr>
<td valign="top"><a target="_blank" rel="noopener noreferrer" href="https://user-images.githubusercontent.com/87758239/164262093-afd7e6a2-759e-4a1a-8a19-c93847339169.png"><img src="https://user-images.githubusercontent.com/87758239/164262093-afd7e6a2-759e-4a1a-8a19-c93847339169.png" width="360" style="max-width: 100%;"></a></td>
<td valign="top"><a target="_blank" rel="noopener noreferrer" href="https://user-images.githubusercontent.com/87758239/164262110-a7959869-7cf3-4060-9a09-e490576c1722.png"><img src="https://user-images.githubusercontent.com/87758239/164262110-a7959869-7cf3-4060-9a09-e490576c1722.png" width="360" style="max-width: 100%;"></a></td>
</tr>
</tbody>
</table>

Also, the back link will now appear on the 'Request sent' page.

So, even though it's a bit repetitive, we should instead include the link in `start_page.html.erb` view.

Reply:
Thanks Jon, the back link is now included in each view where required.